urwid/widget/columns.py

```diff
@@ -1145,7 +1145,7 @@ def get_pref_col(self, size: tuple[()] | tuple[int] | tuple[int, int]) -> int:
             col += sum(widths[: self.focus_position])
         return col
 
-    def rows(self, size: tuple[int] | tuple[int, int], focus: bool = False) -> int:
+    def rows(self, size: tuple[int], focus: bool = False) -> int:
```

Comment:
fix type annotation. While not officially `tuple[int, int]` is accepted, it's incorrect behaviour.